### PR TITLE
Some stuff

### PR DESCRIPTION
**Pull request by Shnupbups**
Back to mapping random things.

### CODE REVIEW

**Review by dblsaiko — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -102,6 +102,8 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 1 world
 		ARG 2 pos
 		ARG 3 side
+	METHOD method_22358 (Lnet/minecraft/class_2680;Lnet/minecraft/class_3611;)Z
```
Comment (ghost) [inline mappings/net/minecraft/block/Block.mapping:106]: method could be called `canFluidBreak` ?
(edit: This is just a suggestion, I do approve the current changes)

File: mappings/net/minecraft/class_4543.mapping [hunk 2]
```diff
@@ -1,3 +1,4 @@
 CLASS net/minecraft/class_4543
 	CLASS class_4544
 		METHOD method_16359 getBiome (III)Lnet/minecraft/class_1959;
```
Comment (liach) [inline mappings/net/minecraft/class_4543.mapping:3]: This is not really getting biome...

**Review by Juuxel — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by liach** [comment]
@Shnupbups Can you update to fix the merge conflicts?

**Review by liach — APPROVED** [review]
@modmuss50 Can you merge this?

**Pull request merged**